content/vendor/cli/release download.md

```diff
@@ -0,0 +1,37 @@
+---
+date: 2021-09-10
+linktitle: "release download"
+title: "release download"
+weight: 90121
+---
+
+Download the YAML config for a release. Same as 'release inspect' for non-KOTS apps.
```

Comment:
Download the YAML config file for a release.

**Note:** This command is the same as the `release inspect` command for non-KOTS apps.